packages/table/src/useLeafyGreenVirtualTable/useLeafyGreenVirtualTable.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react';
+import { useVirtualizer, VirtualItem } from '@tanstack/react-virtual';
+
+import useLeafyGreenTable, { LGRowData } from '../useLeafyGreenTable';
+
+import {
+  LeafyGreenVirtualItem,
+  LeafyGreenVirtualTable,
+  LeafyGreenVirtualTableOptions,
+} from './useLeafyGreenVirtualTable.types';
+
+function useLeafyGreenVirtualTable<
+  T extends LGRowData,
+  V extends unknown = unknown,
+>({
+  containerRef,
+  data,
+  columns,
+  hasSelectableRows,
+  withPagination = false,
+  allowSelectAll = true,
+  virtualizerOptions,
+  ...rest
+}: LeafyGreenVirtualTableOptions<T, V>): LeafyGreenVirtualTable<T> {
+  const table = useLeafyGreenTable({
+    data,
+    columns,
+    withPagination,
+    allowSelectAll,
+    hasSelectableRows,
+    ...rest,
+  });
+
+  const { rows } = table;
```

Comment:
```
const getRows = useCallback(() => table.getRowModel().rows, [table]
```
and then using like this:
```
getRows()
```
causes the table virtual scrolling to get all wonky. I think it as to do with the useCallback.

I think row is supposed to to be called on every hook call because we need the latest row data to pass to the `useVirtualizer` instance